gci/gitorg.py

```diff
@@ -13,11 +14,12 @@
 class GitOrg():
 
     def __init__(self, name):
+        logger = logging.getLogger(__name__ + '__init__')
```

Comment:
you need to add dots to *all* of the loggers.
And please rebase before asking for reviews.

Reply:
sorry, I missed it.

Reply:
done @jayvdb